src/components/content/deploymentVariables/DeploymentVariables.tsx

```diff
@@ -0,0 +1,233 @@
+                return <div className={deploymentVariablesStyles.variablesColumns}>{text}</div>;
+            },
+        },
+        {
+            title: <div className={deploymentVariablesStyles.variablesColumns}>Mandatory</div>,
+            dataIndex: 'mandatory',
+            render: (text: boolean | undefined | null) => {
+                if (text === true) {
+                    return <div className={deploymentVariablesStyles.variablesColumns}>{`true`}</div>;
+                } else if (text === false) {
+                    return <div className={deploymentVariablesStyles.variablesColumns}>{`false`}</div>;
+                } else {
+                    return null;
+                }
+            },
+        },
+        {
+            title: <div className={deploymentVariablesStyles.variablesColumns}>ValueSchema</div>,
+            dataIndex: 'valueSchema',
+            render: (_text: string, record: DeployVariable) => {
+                if (!record.valueSchema) {
+                    return '';
+                }
+
+                const formattedValueSchema = Object.entries(record.valueSchema)
+                    .map(([key, value]) => `${key}: ${value as string}`)
+                    .join('\n');
+
+                return (
+                    <Popover
+                        content={
+                            <pre>
+                                {' '}
+                                <div className={deploymentVariablesStyles.variablesContentHover}>
+                                    {formattedValueSchema}
+                                </div>
+                            </pre>
+                        }
+                        title={'Value Schema'}
+                        trigger='hover'
+                    >
+                        <Button
+                            className={deploymentVariablesStyles.variablesDataHover}
+                            type={'link'}
+                        >{`valueSchema`}</Button>
+                    </Popover>
+                );
+            },
+        },
+        {
+            title: <div className={deploymentVariablesStyles.variablesColumns}>SensitiveScope</div>,
+            dataIndex: 'sensitiveScope',
+            render: (text: 'none' | 'once' | 'always') => {
+                return <div className={deploymentVariablesStyles.variablesColumns}>{text}</div>;
+            },
+        },
+        {
+            title: <div className={deploymentVariablesStyles.variablesColumns}>AutoFill</div>,
+            dataIndex: 'autoFill',
+            children: [
+                {
+                    title: (
+                        <div className={deploymentVariablesStyles.variablesTableChildrenTitle}>
+                            Deploy Resource Kind
+                        </div>
+                    ),
+                    dataIndex: ['autoFill', 'deployResourceKind'],
+                    key: 'deployResourceKind',
+                    render: (
```

Comment:
we must avoid having this list here delcared once more here. Must be used directly from the generated code.

Reply:
![image](https://github.com/user-attachments/assets/9a03a2c3-99eb-4382-a4f8-b6447dd182ad)

![image](https://github.com/user-attachments/assets/8914dc99-2ede-4adf-b7c5-93f4e463a2f5)

fixed